awesome_owl/static/src/todo_list/todo_list.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { Component, useState, useRef, onMounted } from "@odoo/owl";
+import { TodoItem } from "./todo_item";
+import { Todo } from "./todo";
+
+
+export class TodoList extends Component {
+    static template = "awesome_owl.todolist";
+    static components = { TodoItem };
+
+    setup() {
+        this.todos = useState([]);
+        this.myRef = useRef('input');
+        onMounted(() => {
+            this.myRef.el.focus();
+        });
+    }
+
+    addTodo(ev) {
+        if (ev.keyCode === 13 && ev.target.value != "") {
```

Comment:
I think the tutorial makes you do it but magic numbers are bad :(
```suggestion
        if (ev.key === 'Enter' && ev.target.value) {
```